Exchange-Guide.md

```diff
@@ -0,0 +1,196 @@
+# Exchange Guide (WIP)
+
+## Goal
+This guide describes how to setup `xud` with an existing exchange system and how to connect to the XUD Simulation Network, `xud-simnet` for short and do some test trades. `xud` is in alpha stage and the `xud-simnet` aims to give exchange operators a 'feel' of how things will work. `xud` in it's current stage should never be connected to a production system nor be used on mainnet. You'll be using test coins at all times. This guide purposely is kept relatively manual to make it understandable how `xud` functions. We are working on an improved setup using docker, which will also be the preferred way to use `xud` for exchanges in production.
```

Comment:
I was confused by "for short and do some test trades" in the first sentence until I realized the `xud-simnet` was what's short. I'd either add a comma after "for short", or preferably just drop those two words and add a comm after `xud-simnet`.

"This guide purposely is kept relatively manual to make it understandable how `xud` functions" - I'd either drop this sentence or rewrite as "This guide is purposely kept relatively manual to demonstrate how `xud` functions."

Reply:
Done